docs/_pages/contributing.cn.md

```diff
@@ -88,7 +88,11 @@ and more importantly also try to improve the process along the way as well.
 
  - Once you finalize on a project/task, please open a new JIRA or assign an existing one to yourself. 
       - Almost all PRs should be linked to a JIRA. It's always good to have a JIRA upfront to avoid duplicating efforts.
-      - If the changes are minor, then `[MINOR]` prefix can be added to Pull Request title without a JIRA. 
+      - If the changes are minor, then `[MINOR]` prefix can be added to Pull Request title without a JIRA. There are some tips to help to judge **MINOR** Pull Request :
+        - trivial fixes (for example, a typo, a broken link or an obvious error)
```

Comment:
may be we should add IDE suggestions also here?

Reply:
> may be we should add IDE suggestions also here?

What's the meaning of "suggestions" that you mentioned? If you mean our discussion on the thread(`[DISCUSS] Unify Hudi code cleanup and improvement
`) of the mailing list. I would suggest filing jira issues.

Reply:
@yanghua I was referring to intellisense i.e we can mention it like 

- trivial fixes (for example, a typo, a broken link, intellisense or an obvious error)

Let me know if it makes sense.

Reply:
Ok, let me add it as a kind of example.